New durability section - in development (#4677)
EnterpriseDB/docs: product_docs/docs/pgd/5/durability/commit-scopes.mdx

    * `max_commit_delay` (interval) &mdash; Maximum delay that can be injected to commit to try to keep within the lag limits.

!!! Note

> The `CAMO` commit scope kind is mostly syntactic sugar for `GROUP COMMIT (transaction_tracking = true, commit_decision = partner)` with an additional `DEGRADE ON` clause. It's expected that `GROUP COMMIT` will eventually gain the `DEGRADE ON` clause, making `CAMO` syntax deprecated.

Reviewer: I'm unsure if "syntactic sugar" is an appropriate description here. How about "more readable/convenient" or "easier to understand"?

Author: Moved the statement anyway to the CAMO section, removed the future promise and changed to "is mostly an alias for"

> `CAMO` commit scope kind is mostly syntactic sugar for `GROUP COMMIT (transaction_tracking = true, commit_decision = partner)` with additional `DEGRADE ON` clause. It's expected that `GROUP COMMIT` will eventually gain the `DEGRADE ON` clause as well, making `CAMO` syntax deprecated.

Reviewer: ```suggestion
The `CAMO` commit scope kind is mostly syntactic sugar for `GROUP COMMIT (transaction_tracking = true, commit_decision = partner)` with an additional `DEGRADE ON` clause. It's expected that `GROUP COMMIT` will eventually gain the `DEGRADE ON` clause, making `CAMO` syntax deprecated.
```
